components/eamxx/src/physics/nudging/eamxx_nudging_process_interface.cpp

```diff
@@ -118,9 +154,14 @@ void Nudging::initialize_impl (const RunType /* run_type */)
     create_helper_field("p_mid_ext", scalar2d_layout_mid, grid_name, ps);
     auto pmid_ext = get_helper_field("p_mid_ext");
     auto pmid_ext_v = pmid_ext.get_view<Real*,Host>();
-    in_params.set<std::vector<std::string>>("Field Names",{"p_levs"});
-    host_views["p_levs"] = pmid_ext_v;
-    layouts.emplace("p_levs",scalar2d_layout_mid);
+//    in_params.set<std::vector<std::string>>("Field Names",{"p_levs"});
+//    host_views["p_levs"] = pmid_ext_v;
+//    layouts.emplace("p_levs",scalar2d_layout_mid);
+
+    in_params.set<std::vector<std::string>>("Field Names",{"hybm"});
```

Comment:
I don't think we can expect all nudging source files to use the hybrid coordinate system for the vertical pressure profile.  In fact, `p_levs` is used as the variable name for our vertical remapping because we pass explicit pressure levels for that.  

What is the goal with switching to `hybm` here?  I don't see this being used to build the pressure profile using the hybrid coordinate system anywhere, and wouldn't you need surface pressure as well?

If RRM nudging needs to use the hybrid coordinate system I would suggest adding another pressure type option to the nudging interface, in addition to `TIME_DEPENDENT_3D_PROFILE` and `STATIC_1D_VERTICAL_PROFILE`

Reply:
Maybe a name like "TIME_DEPENDENT_3D_HYBRID" for one that uses horizontally varying surface pressure.  Of course you would also need to load `ps` from file.

Reply:
sorry, this is also debug test, I should remove it

Reply:
since I don't have the plev data, so I tested it using hybm as an alternative option.